packages/docs/docs/careplans/clinical-decision-support.md

```diff
@@ -0,0 +1,71 @@
+
+Large language model based clinical decision support tools, as well as tools that use algorithms for risk assessment or triage fall in the predictive clinical decision support category.
+
+Medplum enables [many implementations](/case-studies) with predictive clinical decision support. Per the HTI final ruling, the predictive clinical decision support systems will become regulated in Dec. 2024. Any system certified to g10, b2, f1 or e1 will be required to provide **Insight Reports** as part of maintaining their certification.
+
+The following sections are best practices for preparing for a predictive clinical decision support certification.
+
+### Training Data
+
+Demonstrating which training data was used to train an algorithm, and keeping a record of the versioning on said data is part of the certification process. In the context of a Medplum implementation, be prepared to keep all of your training data in a [Medplum project](/docs/auth/user-management-guide#background-user-model) which will show which data was used to train the models and that the data is updated (feedback loops).
+
+### Code Systems
+
+It is recommended that data is tagged with UMLS code systems, including LOINC, SNOMED and RxNorm. For certification, data must conform to [USCDI profiles](/docs/fhir-datastore/understanding-uscdi-dataclasses). Implementations of predictive clinical decision support sometimes include annotating data with code systems using an algorithm or large language model.
+
+### Insights Reporting
+
+Electronic health records that support predictive clinical decision support will be required to report on the usage of their product. Prepare the following basic statistics as part of certification: the number of times the decision support was used, how many unique clinicians used it, the number of times it was updated and any complaints received.
+
+## Linked Referential
+
+Linked referential clinical decision support are hyperlinks that link to reference material that is specific to the clinical context of a specific patient or population. The ONC criteria that define this standard are 170.205(a)(3,4) amd relate to the retrieving context-aware knowledge using the HL7 Infobutton.
+
+'''caution
+Medplum is not certified for (a)(3,4) but serves as a basis for those who wish to implement. The Clinical Profile and [Diagnostic Report](https://storybook.medplum.com/?path=/story/medplum-diagnosticreportdisplay--simple) react components serve as common launch points for Infobutton implementations.
+'''
+
+To support the linked referential clinical decision support the system should be capable of retrieving information based on one or more of the following data elements. The Clinical Profile react component highlights the data elements.
+
+TODO: Link to the [Clinical Profile](https://github.com/medplum/medplum/issues/3625) react component
```

Comment:
Let's remove this, but add a comment to issue #3625 to update this doc when the component is finished?